web/profiles/custom/os2loop/themes/os2loop_theme/assets/colors.scss

```diff
@@ -27,12 +27,16 @@ $delete: $red;
 $upvote: $primary-color;
 $follow-content: $primary-color;
 $edit: $secondary-color;
+$subscribe: $primary-color;
+$favourite: $primary-color;
 
 $dark-icon-base-color: darken($dark-grey, 10%);
 $dark-delete: darken($red, 10%);
 $dark-upvote: darken($primary-color, 10%);
 $dark-follow-content: darken($primary-color, 10%);
 $dark-edit: darken($secondary-color, 10%);
+$dark-subscribe: darken($primary-color, 10%);
+$dark-favourite: darken($primary-color, 10%);
```

Comment:
This should be `darken($favourite, 10%);` (in case `$favourite` is set to something other than `$primary-color`).